python/src/MultigroupTable.python.cpp

```diff
@@ -36,11 +37,13 @@ void wrapMultigroupTable( python::module& module, python::module& ) {
     python::init< std::string, std::string, std::string, std::string,
                   double, double, double, double,
                   EnergyGroupStructure, FluxWeights,
-                  ReactionCrossSections >(),
+                  ReactionCrossSections,
+                  std::optional< AverageFissionEnergyRelease > >(),
     python::arg( "zaid" ), python::arg( "libname" ), python::arg( "source" ),
     python::arg( "process" ), python::arg( "awr" ), python::arg( "weight" ),
     python::arg( "temperature" ), python::arg( "dilution" ),
     python::arg( "structure" ), python::arg( "flux" ), python::arg( "xs" ),
+    python::arg( "release" ) = std::nullopt,
```

Comment:
Could you name this fiss_q to be consistent with the original NDI files?

Reply:
I'll change it to fission_q